source/api/tests/requirements-test.txt

```diff
@@ -2,5 +2,8 @@ aws_xray_sdk
 awswrangler
 boto3
 chalice
+moto==4.1.2
```

Comment:
Our appsec guidance generally recommends against pegging versions. Double check that latest won't work for moto.

Reply:
Its used in data-connectors
https://github.com/aws-solutions/data-connectors-for-aws-clean-rooms/blob/main/source/requirements-dev.txt#L12